cpp/include/cudf/detail/aggregation/aggregation.hpp

```diff
@@ -62,37 +62,6 @@ struct quantile_aggregation : aggregation {
   experimental::interpolation _interpolation;  ///< Desired interpolation
 };
 
-/**
- * @brief Maps an `aggregation::Kind` value to it's corresponding binary
- * operator.
- *
- * @note Not all values of `aggregation::Kind` have a valid corresponding binary
- * operator. For these values `E`,
- * `std::is_same_v<corresponding_operator<E>::type, void>`.
- *
- * @tparam k The `aggregation::Kind` value to map to its corresponding operator
- */
-template <aggregation::Kind k>
-struct corresponding_operator {
-    using type = void;
-};
-
-template <>
-struct corresponding_operator<aggregation::MIN> {
-  using type = DeviceMin;
-};
-template <>
-struct corresponding_operator<aggregation::MAX> {
-  using type = DeviceMax;
-};
-template <>
-struct corresponding_operator<aggregation::SUM> {
-  using type = DeviceSum;
-};
```

Comment:
Hey, wait. I was using that. https://github.com/rapidsai/cudf/pull/3597/files?file-filters%5B%5D=.cu&file-filters%5B%5D=.cuh#diff-5e6a81797c607512482b2ac74bab8435R56

Reply:
Ah, okay, I'll restore it.